app/javascript/packs/base.jsx

```diff
@@ -96,15 +96,17 @@ function initializeNav() {
       '.js-hamburger-trigger, .hamburger a:not(.js-nav-more-trigger)',
     ),
   ];
-  const memberMenu = document.getElementById('crayons-header__menu');
-  const menuNavButton = document.getElementById('member-menu-button');
   const moreMenus = [...document.getElementsByClassName('js-nav-more-trigger')];
 
   setCurrentPageIconLink(currentPage, getPageEntries());
   initializeMobileMenu(menuTriggers, moreMenus);
-  initializeTouchDevice(memberMenu, menuNavButton);
 }
 
+const memberMenu = document.getElementById('crayons-header__menu');
```

Comment:
So the fix is to only run `initializeTouchDevice` when the page loads. When it was always being  loaded via InstantClick page changes, it kept adding event listeners each time to the member navigation button which is what caused the bug and also was attaching _n_ events where the number of pages navigated to was _n_.

Reply:
we have this in `initializeTouchDevice`:

```js
// Use a named function instead of anonymous so duplicate event handlers are discarded
        menuNavButton.addEventListener('click', (_event) => {
          toggleHeaderMenu(memberTopMenu, menuNavButton);
        });
```

I guess it wasn't ignoring duplicates as we _are_ using an anonymous function as the event listener. In any case, the way you've fixed it makes more sense to me, by only initializing once. But let's remove that comment on line 62 as well?